packages/auth/src/grant/routes.ts

```diff
@@ -384,12 +411,37 @@ async function continueGrant(
   const access = await accessService.getByGrant(grant.id)
 
   // TODO: add "continue" to response if additional grant request steps are added
-  ctx.body = {
+  ctx.body = createGrantBody({
+    domain: config.authServerDomain,
+    grant,
+    access,
+    accessToken
+  })
+}
+
+function createGrantBody({
+  domain,
+  grant,
+  access,
+  accessToken
+}: {
+  domain: string
+  grant: Grant
+  access: Access[]
+  accessToken: AccessToken
+}) {
+  return {
     access_token: {
       value: accessToken.value,
-      manage: config.authServerDomain + `/token/${accessToken.managementId}`,
+      manage: domain + `/token/${accessToken.managementId}`,
       access: access.map((a: Access) => accessToBody(a)),
       expiresIn: accessToken.expiresIn
+    },
+    continue: {
+      access_token: {
+        value: grant.continueToken
+      },
+      uri: domain + `continue/${grant.continueId}`
```

Comment:
Should this still include `wait` (`RECOMMENDED`)?
https://datatracker.ietf.org/doc/html/draft-ietf-gnap-core-protocol#section-3.1

Reply:
I don't think so because the grant is granted right away. At least that was my interpretation of it.